operator/docs/dev/security/results.golint.txt

```diff
@@ -23,17 +21,17 @@ controllers/internal/csiscaleoperator/csiscaleoperator.go:113:1: exported method
 controllers/internal/csiscaleoperator/csiscaleoperator.go:117:1: exported method CSIScaleOperator.GetCSIControllerPodLabels should have comment or be unexported
 controllers/internal/csiscaleoperator/csiscaleoperator.go:121:1: exported method CSIScaleOperator.GetCSINodePodLabels should have comment or be unexported
 controllers/internal/csiscaleoperator/csiscaleoperator.go:125:1: exported method CSIScaleOperator.GetDefaultImage should have comment or be unexported
-controllers/internal/csiscaleoperator/csiscaleoperator.go:151:1: exported method CSIScaleOperator.GetKubeletRootDirPath should have comment or be unexported
-controllers/internal/csiscaleoperator/csiscaleoperator.go:162:1: exported method CSIScaleOperator.GetSocketPath should have comment or be unexported
-controllers/internal/csiscaleoperator/csiscaleoperator.go:170:1: exported method CSIScaleOperator.GetSocketDir should have comment or be unexported
-controllers/internal/csiscaleoperator/csiscaleoperator.go:178:1: exported method CSIScaleOperator.GetCSIEndpoint should have comment or be unexported
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:33:2: const snapshotStorageApiGroup should be snapshotStorageAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:34:2: const securityOpenshiftApiGroup should be securityOpenshiftAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:35:2: const storageApiGroup should be storageAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:36:2: const rbacAuthorizationApiGroup should be rbacAuthorizationAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:37:2: const coordinationApiGroup should be coordinationAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:38:2: const podSecurityPolicyApiGroup should be podSecurityPolicyAPIGroup
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:249:1: comment on exported method CSIScaleOperator.GenerateProvisionerClusterRoleBinding should be of the form "GenerateProvisionerClusterRoleBinding ..."
-controllers/internal/csiscaleoperator/csiscaleoperator_package.go:643:1: receiver name s should be consistent with previous receiver name c for CSIScaleOperator
-controllers/syncer/csi_node.go:58:2: exported const EnvVarForCSINodeRegistrarImage should have comment (or a comment on this block) or be unexported
+controllers/internal/csiscaleoperator/csiscaleoperator.go:151:1: exported method CSIScaleOperator.GetKubeletPodsDir should have comment or be unexported
```

Comment:
This is new. Can we add a docstring for this method `GetKubeletPodsDir` to remove this lint message.

Reply:
Ideally we should unexport all these functions by starting these with small letters (as these are not exposed outside), but we can add a docstring for now to avoid this new warning.